scripts/mkc.sh

```diff
@@ -5,31 +5,23 @@ CERT_PATH=/Users/xcode/certificates.p12
 
 echo "Keychain Starting"
 
-# Code 0 = Ok
-# Code 1 = Locked
-# Code 36 = Locked
-# Code 50 = Not Found
-/usr/bin/security show-keychain-info $KC_NAME 2>/dev/null
+# check if target keychain exists:
+/usr/bin/security list-keychains -d user | grep $KC_NAME &> /dev/null
```

Comment:
@jleach see here: instead of `security show-keychain-info` it's updated to `security list-keychains`. I found the issue where even this is running in the background, of the keychain is not yet unlocked, it'll prompt a window for password entry. That explains why we are seeing that from time to time on agent and also why cicd keychain is not unlocked. So there's no return for the Code value anymore, I only check if it exists or not, if yes, always unlock it. If not the create.